### PR TITLE
fix for war-2104(selenium test cases are failing in travis)

### PR DESCRIPTION
**Pull request by apathapa-warrior**

### CODE REVIEW

File: wftests/warrior_tests/testcases/selenium_func_tests/generic_tests/tc_selenium_generic_10.xml [hunk 1]
```diff
@@ -29,15 +29,15 @@
...
-		<step TS= '3' Driver='selenium_driver' Keyword='verify_text' >
-			<Arguments>
-				<argument name="system_name" value="search_element"/>
-                <argument name="locator" value="id-search-field"/>
-				<argument name="locator_type" value="id"/>
-				<argument name="var" value="searchtext"/>
-				<argument name="expected" value="Search"/>
-			</Arguments>
-		</step>
+		<!--<step TS= '3' Driver='selenium_driver' Keyword='verify_text' >-->
```
Comment (sanikakulkarni) [inline wftests/warrior_tests/testcases/selenium_func_tests/generic_tests/tc_selenium_generic_10.xml]: Hi @apathapa-warrior,

Is there a reason why this step was commented out?

Comment (sanikakulkarni) [reply wftests/warrior_tests/testcases/selenium_func_tests/generic_tests/tc_selenium_generic_10.xml]: Looks like the corresponding py2 PR was merged. I will create a ticket to track this commented out step.

Comment (sanikakulkarni) [reply wftests/warrior_tests/testcases/selenium_func_tests/generic_tests/tc_selenium_generic_10.xml]: Created WAR-2111 to track this. Resolving comments.

**Comment by apathapa-warrior** [comment]
In removed step expected string for the value attribute is "Search", but in python.org now the value is '' (empty), element info given below. As verify_text keyword is already present in other steps, we removed this step.

<input id="id-search-field" name="q" type="search" role="textbox" class="search-field" placeholder="Search" value="" tabindex="1">

**Pull request merged**